papers/ring_vrf/overview.tex

```diff
@@ -1,23 +1,31 @@
-\section{Protocol overview}
+\section{An Overview on Ring VRFs}
 \label{sec:overview}
 
 As a beginning, we introduce the ring VRF interface, give a simple
-unamortized NIZK that realizes the ring VRF properties discussed later,
-and give some intuition for our later amortization trick.
+unamortized non-interactive zero-knowledge (NIZK) protocol that realizes the ring VRF properties discussed formally in \S\ref{subsec:uc_mode},
+and give some intuition for our later amortization trick. 
 
-Any VRF or ring VRF needs a \KeyGen algorithm, which returns a secret key \sk
-and associated public key \pk, as well as an evaluation algorithm
-$\Eval : (\sk,\msg) \mapsto \Out$. % for which many constructions exist.
+
+As VRF \cite{vrf_micali}, a ring VRF needs a \KeyGen algorithm and an $ \Eval $ algorithm,
+
+\begin{itemize}
+	\item $ \rVRF.\KeyGen: (\secparam) \mapsto (\sk, \pk) $  returns a secret key \sk and
+	associated public key \pk given the security parameter $ \secparam $.
```

Comment:
\secparam is declared to be implicit everywhere later, no point in discussing it here.  I'll display Eval like you suggest, as it deserves more comments, like the cites you add, but we've nothing to say about KeyGen here.